person_capture/gui_app.py

```
@@ -10722,6 +10722,179 @@ def _save_hdr_archive_crop(
+                    except Exception:
+                        pass
+                if os.path.exists(ref_out_path):
+                    os.replace(ref_out_path, ref_backup)
+                    backed_up_ref = True
+                if os.path.exists(clean_out_path):
+                    os.replace(clean_out_path, clean_backup)
+                    backed_up_clean = True
+
+                os.replace(ref_tmp, ref_out_path)
+                os.replace(clean_tmp, clean_out_path)
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)